vxquery-xtest/src/main/java/org/apache/vxquery/xtest/XTest.java

```diff
@@ -100,6 +100,13 @@ synchronized void waitForCompletion() throws InterruptedException {
             eSvc.awaitTermination(opts.keepalive, TimeUnit.MILLISECONDS);
         } finally {
             try {
+
+                //Perform the sorting process.
+                if (opts.diffable != null) {
```

Comment:
Instead of altering this file, can you move this change to LineFileReporterImpl.close()